pymeasure/instruments/attocube/adapters.py

```diff
@@ -25,7 +25,31 @@
 import re
 import time
 
-from pymeasure.adapters import TelnetAdapter
+from pymeasure.adapters import ProtocolAdapter, TelnetAdapter
+
+
+class Mock_TelnetAdapter(ProtocolAdapter):
```

Comment:
Please keep test-specific code in the tests/ directory (in this case test_anc300.py), to keep that out of the pymeasure installed package.